debian/control

```diff
@@ -36,7 +36,7 @@ Vcs-Browser: https://github.com/tseliot/ubuntu-drivers-common
 X-Python3-Version: >= 3.2
 
 Package: ubuntu-drivers-common
-Architecture: any
+Architecture: amd64
```

Comment:
Can these arch changes be undone now?

Reply:
I'll rebase this and try dropping the changes.